src/core/api_client/tests/factories/mock.factory.ts

```
@@ -0,0 +1,314 @@
+        code_challenge_methods_supported: ['plain', 'S256'],
+        tls_client_certificate_bound_access_tokens: true,
+        backchannel_logout_supported: true,
+        backchannel_logout_session_supported: true,
+        authorization_response_iss_parameter_supported: true,
+      },
+    });
+  }
+
+  static setupApiEndpoint(path: string, response: any = {}): void {
+    fetchMock.post(AuthFactory.getApiUrl(path), (url: string, opts: any) => {
+      this.callCount++;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)